contracts/WinnablesPrizeManager.sol

```diff
@@ -222,6 +223,10 @@ contract WinnablesPrizeManager is Roles, BaseCCIPSender, BaseCCIPReceiver, IWinn
     /// @param tokenId ID of the NFT
     function withdrawNFT(address nft, uint256 tokenId) external onlyRole(0) {
         if (_nftLocked[nft][tokenId]) revert NFTLocked();
+
+        try ERC721(nft).tokenURI(tokenId) returns (string memory) {} catch {
```

Comment:
use `ERC165(nft).supportsInterface()` instead. NFTs have to support interface `type(IERC721).interfaceId`